frontend/packages/console-shared/src/components/drawer/Drawer.scss

```diff
@@ -0,0 +1,43 @@
+.ocs-drawer {
+  position: relative;
+  visibility: hidden;
+  overflow-x: hidden;
+  overflow-y: hidden;
+  width: 100%;
+  flex-grow: 0;
+  flex-shrink: 0;
+  transition: opacity 175ms ease-out, transform 225ms ease-out;
+  border-top: 1px solid var(--pf-global--palette--black-400);
+
+  &__drag-handle {
+    width: 100%;
+    height: 6px;
+    cursor: ns-resize;
+    position: absolute;
+    background-color: var(--pf-global--palette--black-200);
```

Comment:
This should be transparent with a hover state that changes the background so the user has another visual cue they are over the drag handle.